### DISCUSSION TITLE
Welcome to evSeq Discussions!

### DISCUSSION THREAD

**Discussion by brucejwittmann** [fhalab/evSeq#30] [Announcements]
This is a place for discussing non-software-related questions/problems with the evSeq workflow!

**Comment by YTMiiikey** [comment]
Hi! I have a question. I noticed that the full PCR/gel extraction protocol mentioned that it is critical to use a loading dye with no SDS when running the DNA gel. Why is that? I know that SDS could denature proteins in the PCR reaction, but I don't understand how that would affect the result here. Thanks!

**Reply by brucejwittmann** [reply]
If you are following the protocol exactly, it is because the gel used to separate evSeq libraries from primer dimers contains SYBR Gold. Loading dyes with SDS in them can interfere with band migration when using SYBR as a gel stain. Usually this isn't something that is noticed, but when trying to separate small bands it can result in severe smearing of the product. When I've tried to run the protocol using loading dye with SDS in it, it can cause such bad smearing that the library is either unseparable from the primer dimer or make it appear as though the PCR failed entirely.

**Reply by YTMiiikey** [reply]
I see. Thanks for the explanation!
